src/website/views/getting-started/index.pug

```diff
@@ -36,7 +36,14 @@ p.-text
 
 .-mb--2
   :code(lang='bash')
-    $ scripts/docker.sh start
+    scripts/docker.sh alias
+    source ~/.bash_profile
+
+  | Now your environment it's set up. To start development run:
```

Comment:
@HectorGarciaAlvarez this one should render as a paragraph in 14px. Right now, it is included inside the code block and is rendering at 16px. Please add p.-text and remove tabulation so it is indented properly